### ISSUE TITLE
a11y help refers to wrong command name

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#208709]
Testing #207637

In the screenshot below, "Chat: Insert at Cursor" should be "Chat: Insert into New File"

![Image](https://github.com/microsoft/vscode/assets/30305945/9f73b60b-8e70-4f94-bb73-754b5297c0d3)
